src/Core/Services/Implementations/UserService.cs

```diff
@@ -586,45 +584,35 @@ public async Task<CredentialCreateOptions> StartWebAuthnLoginRegistrationAsync(U
         return true;
     }
 
-    public async Task<AssertionOptions> StartWebAuthnLoginAssertionAsync(User user)
+    public Task<AssertionOptions> StartWebAuthnLoginAssertionAsync()
     {
-        var provider = user.GetTwoFactorProvider(TwoFactorProviderType.WebAuthn);
-        var existingKeys = await _webAuthnCredentialRepository.GetManyByUserIdAsync(user.Id);
-        var existingCredentials = existingKeys
-            .Select(k => new PublicKeyCredentialDescriptor(CoreHelpers.Base64UrlDecode(k.CredentialId)))
-            .ToList();
+        var options = _fido2.GetAssertionOptions(Enumerable.Empty<PublicKeyCredentialDescriptor>(), UserVerificationRequirement.Required);
 
-        if (existingCredentials.Count == 0)
+        return Task.FromResult(options);
+    }
```

Comment:
I don't believe there are multiple implementations of `IUserService` so this method shouldn't have to remain task returning and can become sync.

Reply:
Done!